packages/vx-demo/src/docs-v2/examples/vx-glyph/Example.tsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable jsx-a11y/accessible-emoji */
+import React from 'react';
+import { Group } from '@vx/group';
+import {
+  Glyph as CustomGlyph,
+  GlyphCircle,
+  GlyphCross,
+  GlyphDiamond,
+  GlyphSquare,
+  GlyphStar,
+  GlyphTriangle,
+  GlyphWye,
+} from '@vx/glyph';
+import { LinePath } from '@vx/shape';
+import genDateValue, { DateValue } from '@vx/mock-data/lib/generators/genDateValue';
+import { scaleTime, scaleLinear } from '@vx/scale';
+import { curveMonotoneX, curveBasis } from '@vx/curve';
+
+const defaultMargin = { top: 10, right: 10, bottom: 10, left: 10 };
+
+// colors
+export const primary = '#8921e0';
+const secondary = '#00f2ff';
+const contrast = '#ffffff';
+
+// Glyphs to render
+const Glyphs = [
```

Comment:
refactored from just showing the `GlyphDot` glyph (omitted it because it's the only one that uses `r` + `cx` + `cy` instead of `size` + `left` + `top`)